README.md

```diff
@@ -23,7 +23,7 @@ Each file contains:
 | **Variables**                              |                                                                             |
 | Create variables                           | [creating-variables.js](2-variables/creating-variables.js)                  |
 | JavaScript Numbers                         | [numbers.js](2-variables/numbers.js)                                        |
-| Booleans in JavaScript                     | [booleans.js](2-variables/booleans.js)                                        |
+| Booleans in JavaScript                     | [booleans.js](2-variables/booleans.js)                                      |
```

Comment:
What did you changed here? Normally no need for a change

Reply:
I removed a empty space, for a consistent format.